gradle-enterprise-conventions-gradle-plugin/example-builds/gradle_5.6/settings.gradle

```diff
@@ -0,0 +1,9 @@
+pluginManagement {
+ repositories {
+     mavenLocal()
```

Comment:
To be replaced with project local repo.

mavenLocal() should generally be avoided given that it is unmanaged state.

Reply:
Changed me mind. Let's go with this as it's familiar to people.